src/expr/mod.rs

```diff
@@ -519,6 +519,20 @@ impl Expression {
 
         result_stack.pop().unwrap()
     }
+
+    /// The original string which has been parsed to produce this ['Expression`].
+    ///
+    /// ```
+    /// use cfg_expr::Expression;
+    ///
+    /// assert_eq!(
+    ///     Expression::parse("any()").unwrap().original(),
+    ///     "any()"
+    /// );
+    /// ```
+    pub fn original(&self) -> &str {
```

Comment:
```suggestion
    /// ```
    #[inline]
    pub fn original(&self) -> &str {
```